.github/workflows/update-chart.yml

```diff
@@ -4,9 +4,9 @@ name: Update Chart
 
 # Controls when the action will run. 
 on:
-  # Runs every day
-  schedule:
-    - cron: "0 0 * * *"
+  # Runs every day (Temporarily disabled)
+  # schedule:
+  #   - cron: "0 0 * * *"
```

Comment:
You can disable a workflow in the Actions tab without needing to modify its yaml